tiledb/ml/readers/pytorch.py

```diff
@@ -69,50 +56,62 @@ def PyTorchTileDBDataLoader(
     if not all(key_range.equal_values(schema.key_range) for schema in schemas[1:]):
         raise ValueError(f"All arrays must have the same key range: {key_range}")
 
-    if kwargs.get("num_workers") and any(
-        schema.kind is not TensorKind.DENSE for schema in schemas
-    ):
-        raise NotImplementedError("https://github.com/pytorch/pytorch/issues/20248")
+    datapipe_for_key_range = partial(_get_datapipe, schemas)
```

Comment:
Please add some comments because it's a tricky part.

Reply:
Comments & docstrings added at a93e6752115407c79f409ef296cc4e7f5a6fec38